src/lib/rule-finder.js

```diff
@@ -19,15 +19,29 @@ function _getConfigFile(specifiedFile) {
 }
 
 function _getConfigs(configFile, files) {
-  const cliEngine = new eslint.CLIEngine({
+  if (ESLint) { // ESLint 7+
+    const cliEngine = new ESLint({
+      // Ignore any config applicable depending on the location on the filesystem
+      useEslintrc: false,
+      // Point to the particular config
+      overrideConfigFile: configFile
+    });
+
+    return new Set(files
+      .map(filePath => cliEngine.isPathIgnored(filePath) ? false : cliEngine.calculateConfigForFile(filePath))
+      .filter(Boolean));
```

Comment:
it'd be nice if we could avoid repeating these lines, since the only difference is the "get config" method.

Reply:
Also using `calculateConfigForFile` instead of `getConfigForFile` in v7+